scripts/tools/IndexSummary.php

```diff
@@ -68,15 +68,38 @@ protected function run(): Report
             $report = Report::createInfo($data['label']);
             $report->add(Report::createSuccess('Total in DB: ' . $data['totalInDb']));
             $report->add(Report::createSuccess('Total indexed "' . $data['index'] . '": ' . $totalIndexed));
-            $report->add(new Report($percentage < 100 ? Report::TYPE_ERROR : Report::TYPE_SUCCESS, 'Percentage indexed: ' . $percentage . '%'));
-            $report->add(new Report($missingIndex > 0 ? Report::TYPE_ERROR : Report::TYPE_SUCCESS, 'Missing items: ' . $missingIndex));
+            $report->add(
+                new Report(
+                    $this->getPercentageReportTypeBy($percentage, $totalIndexed),
+                    'Percentage indexed: ' . $percentage . '%'
+                )
+            );
+            $report->add(
+                new Report(
+                    $missingIndex > 0 ? Report::TYPE_ERROR : Report::TYPE_SUCCESS,
+                    'Missing items: ' . $missingIndex
+                )
+            );
 
             $mainReport->add($report);
         }
 
         return $mainReport;
     }
 
+    private function getPercentageReportTypeBy(float $percentage, int $totalIndexed): string
+    {
+        if ($totalIndexed == 0) {
```

Comment:
Why 0 indexed is a warning now?

```suggestion
        if ($totalIndexed === 0) {
```

Reply:
fixed, the intention was to not show percentage red even if it is 0% because if there is nothing to index it is not an error.

Reply:
Thanks! 

In this case we also need to make sure the number of records in the DB is also 0, no?